index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css"/>
+    <title>My Portfolio</title>
+</head>
+<body>
+ <div class="mobile-finish">
+   <section class="full-bgcolor">
+    <header class="header">
+        <nav class="navbar">
+           <a class="header-logo" href="#">My Logo</a>
+           <div class="menu-icon">
+           <a class="menubar" href="#"><img src="/images/Enabled.png" alt="menu" /></a>
+           </div>
+        </nav>
+    </header>
+    <main class="main-content">
+        <section class="headline">
+            <h1 class="headline-title">I am Md Ariful Glad to see you!</h1>
+            <p class="headline-content">I'm a software developer I can help you
+                build a product feature or website Look through some of my works and
+                experience! If you like what you see and have a project you need coded
+                don't hesitate to contact me.
+            </p>
+         <div class="headline-social">
+            <h3 class="social-title">LET'S CONNECT</h3>
+            <ul class="social-links">
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/twitter.svg" alt="Twiter logo"/></a>
+                </li>
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/Linkedin icon.svg" alt="Linkdin logo"/></a>
+                </li>
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/medium.svg" alt="Medium logo" /></a>
+                </li>
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/github.svg" alt="Github Logo" /></a>
```

Comment:
Please make sure to remove the template logo. Additionally yo can replace it with your own logo if you have one.